requirements/all.txt

```diff
@@ -358,3 +217,208 @@ numpy==1.15.1 \
+# Code: https://github.com/getsentry/raven-python
+raven==6.1.0 \
+    --hash=sha256:02cabffb173b99d860a95d4908e8b1864aad1b8452146e13fd7e212aa576a884 \
+    --hash=sha256:56dc9062dd42bca97350e5048ff417c914376366caa3b1b5f788b27ddc0a34b7
+
+# Python client for Redis
+# TODO: redis or hiredis or both?
+# Code: https://github.com/andymccurdy/redis-py
+redis==2.10.5 \
+    --hash=sha256:5dfbae6acfc54edf0a7a415b99e0b21c0a3c27a7f787b292eea727b1facc5533 \
+    --hash=sha256:97156b37d7cda4e7d8658be1148c983984e1a975090ba458cc7e244025191dbd
+
+# Python HTTP Requests for Humans™
+# Code: https://github.com/psf/requests
+requests[security]==2.18.3 \
+    --hash=sha256:b62be4ec5999c24d10c98d248a136e7db20ca6616a2b65060cd9399417331e8a \
+    --hash=sha256:fb68a7baef4965c12d9cd67c0f5a46e6e28be3d8c7b6910c758fbcc99880b518
+
+# Mocked responses for the requests library
+# Code: https://github.com/jamielennox/requests-mock
+requests-mock==1.3.0 \
+    --hash=sha256:23edd6f7926aa13b88bf79cb467632ba2dd5a253034e9f41563f60ed305620c7 \
+    --hash=sha256:bd86970d6c52cc97071f5185aa594de6a997a5ca63b3bb36aceb9bb9db49294b
+
+# R-Tree spatial index for Python GIS, requires libspatialindex
+# Code: https://github.com/Toblerity/Rtree
+Rtree==0.8.3 \
+    --hash=sha256:6cb9cf3000963ea6a3db777a597baee2bc55c4fc891e4f1967f262cc96148649 \
+    --hash=sha256:8526431aa15d8cea1c07b451ce853b62b1da4cf6b74c07810aba5b81d12efe66 \
+    --hash=sha256:a9e67386073f93f22449f396d5993dfe479335169c376f3f7fb04a396391f0dc
+
+# Toolkit for mathematics, science, and engineering
+scipy==0.19.1 \
+    --hash=sha256:023ee29faa76c184a607e21076f097dc32f3abba7c71ece374588f95920aa993 \
+    --hash=sha256:1e8fedf602859b541ebae78667ccfc53158edef58d9ee19ee659309004565952 \
+    --hash=sha256:2a26d06a642e3c9107ca06df125f5dc5507abe2b87fd7865415d03ab654b0b43 \
+    --hash=sha256:2a3b6ceadbb58d8b8d4a329f8219f9e6f17757ec6c85baf03987bbd2b728c263 \
+    --hash=sha256:2c9e87d556b83a8e11de7a064856c3997bbaff7f9cf62bf63ff0623751549e4c \
+    --hash=sha256:2dc9bcfaa585d9d941fb1add0d160556fe8587c3800264a77643695565a2d279 \
+    --hash=sha256:4118647273c907ed984da52b71fa2bbb30229f1820fb79b1ed087c8bc9a20cca \
+    --hash=sha256:4c26ef44e8bb2cd2aef11c60d163caa04670d6f42996789b209526677310ded2 \
+    --hash=sha256:4de84e3870228e8afd55a6e63e762aa7c9d1e3bd9a9ef5ab716835a69c77d257 \
+    --hash=sha256:78713101b32af384c564837fd7ae665a2a72cb6d49edbd8f32148d74724a65a8 \
+    --hash=sha256:909be30bdd1d8cd57609760ea72a1499543169ed6ea5c66fad50946582682cf8 \
+    --hash=sha256:93825cc80c638d901099f657dfff852ad2421beb51cb7d1d3f91157741ebe287 \
+    --hash=sha256:a19a2ca7a7336495ec180adeaa0dfdcf41e96dbbee90d51c3ed828ba570884e6 \
+    --hash=sha256:b3e97be2cd9f052d984fc5ba2d441897971b744c64d658617944c47bc366f8ff \
+    --hash=sha256:b87a923ed390ba0aafcbbcc6f0e023d495f19d2bd22ae59bdef3b0aec6485b39
+
+# Customize the process title
+# https://github.com/dvarrazzo/py-setproctitle
+setproctitle==1.1.10 \
```

Comment:
``setproctitle`` may be unused

Reply:
This is an optional dependency of gunicorn: https://github.com/benoitc/gunicorn/blob/0ce857fced9e8a66c05014054821ace9c56681b9/docs/source/faq.rst

Reply:
Interesting, yes we want to keep that. It looks like gunicorn also has ``gunicorn[gevent]`` in the current release, but not our version.